starknet_devnet/starknet_wrapper.py

```diff
@@ -198,9 +198,10 @@ def get_transaction_status(self, transaction_hash: str):
             if "block_hash" in transaction:
                 ret["block_hash"] = transaction["block_hash"]
 
-            failure_key = "tx_failure_reason"
-            if failure_key in transaction:
-                ret[failure_key] = transaction[failure_key]
+            curr_failure_key = "transaction_failure_reason"
+            new_failure_key = "tx_failure_reason"
```

Comment:
we can lose `new_failure_key`, just use `"tx_failure_reason"` directly.

Reply:
Also, comment that the `if key in object` is performed to check if the tx really failed and contains a failure reason.